specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2025-02-01/examples/Capabilities.json

```diff
@@ -24,22 +24,24 @@
         "properties": {
           "dnsZones": [
             {
-              "subResource": "AzureBackup",
               "requiredZoneNames": [
                 "privatelink.wus.backup.windowsazure.com",
                 "privatelink.queue.core.windows.net",
                 "privatelink.blob.core.windows.net"
-              ]
+              ],
+              "subResource": "AzureBackup"
             },
             {
-              "subResource": "AzureSiteRecovery",
               "requiredZoneNames": [
                 "privatelink.siterecovery.windowsazure.com"
-              ]
+              ],
+              "subResource": "AzureSiteRecovery"
             }
           ]
         }
       }
     }
-  }
+  },
+  "operationId": "RecoveryServices_Capabilities",
+  "title": "Capabilities for Microsoft.RecoveryServices/Vaults"
```

Comment:
won't people look for these at the top usually?

Reply:
Look for what? Could you elaborate?

Reply:
@pshao25 I mean, since operationId and title are good 'context' in which to undertstand the rest of the document, shouldn't they be positionally placed at the top, so that people read them first?